<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="txtMaDoanVien">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Mã đoàn viên</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>51</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string>Quản Lý Đoàn Viên</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtTen">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tên đoàn viên</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbbChucVu">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Đoàn viên</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Phó bí thư</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bí thư</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chức vụ:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Xác nhận</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ngày sinh:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="NgaySinh">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>9999</year>
     <month>11</month>
     <day>30</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Giới tính:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="GioiTinh">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nam</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nữ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Khác</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="txtChiDoan">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Chi đoàn</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtKhoaHoc">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Khóa học</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtKhoa">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Khoa</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtSDT">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Số điện thoại</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtDiaChi">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Địa chỉ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
